satpy/readers/viirs_compact.py

```diff
@@ -99,40 +99,35 @@ def __init__(self, filename, filename_info, filetype_info):
                                or (max(abs(self.min_lat), abs(self.max_lat)) > 60))
 
         self.scans = self.h5f["All_Data"]["NumberOfScans"][0]
-        self.geostuff = self.h5f["All_Data"]['VIIRS-%s-GEO_All' % self.ch_type]
+        self.geography = self.h5f["All_Data"]['VIIRS-%s-GEO_All' % self.ch_type]
 
         for key in self.h5f["All_Data"].keys():
             if key.startswith("VIIRS") and key.endswith("SDR_All"):
                 channel = key.split('-')[1]
                 break
 
-        # FIXME:  this supposes  there is  only one  tiepoint zone  in the
-        # track direction
-        self.scan_size = self.h5f["All_Data/VIIRS-%s-SDR_All" %
-                                  channel].attrs["TiePointZoneSizeTrack"].item()
-        self.track_offset = self.h5f["All_Data/VIIRS-%s-SDR_All" %
-                                     channel].attrs["PixelOffsetTrack"]
-        self.scan_offset = self.h5f["All_Data/VIIRS-%s-SDR_All" %
-                                    channel].attrs["PixelOffsetScan"]
+        # This supposes  there is  only one  tiepoint zone  in the track direction
+        channel_path = "All_Data/VIIRS-%s-SDR_All" % channel
```

Comment:
Should we start using f-strings? `channel_path = f"All_Data/VIIRS-{channel}-SDR_All"`

Reply:
we could, fstrings are included in python >= 3.6